packages/web-components/src/components/content-switcher/content-switcher.ts

```diff
@@ -164,14 +175,17 @@ export default class CDSContentSwitcher extends LitElement {
    * @param direction `-1` to navigate backward, `1` to navigate forward.
    */
   protected _navigate(direction: number) {
-    const { selectorItemSelected } = this
+    const { selectorItemSelected, selectorItemFocused } = this
```

Comment:
```suggestion
    const {selectorItemFocused } = this
```
looks like selectorItemSelected is no longer being used which is causing all the checks to run forever

Reply:
Ohh! Thanks for catching that!